docs/application/dotnet/guides/multimedia/camera.md

```diff
@@ -327,17 +326,16 @@ To customize the display settings:
 
     The system applies display flip after display rotation, so you must always calculate the correct display flip value after determining the display rotation.
 
-> **Note**
->
-> For an overlay surface, when the device orientation changes, the displayed camera preview does not rotate automatically. If you want to rotate the display according to the device orientation, use the `Rotation` property of the `Tizen.Multimedia.CameraDisplaySettings` class.  
+> [!NOTE]
+> For an overlay surface, when the device orientation changes, the displayed camera preview does not rotate automatically. If you want to rotate the display according to the device orientation, use the `Rotation` property of the `Tizen.Multimedia.CameraDisplaySettings` class.
 >
 > For an Evas surface, the Evas object for the camera display is rotated by the window manager used by the application, not by the `Rotation` property.
 
 ## Taking a Photo
```

Comment:
Gerund forms should be avoided for headings.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#verb-form
```suggestion
## Take a photo
```